docker-compose.yml

```diff
@@ -24,6 +24,15 @@ services:
       POSTGRES_PASSWORD: ${POSTGRES_TEST_PASSWORD:-postgres}
     ports:
       - '${POSTGRES_TEST_PORT:-5433}:5432'
+    volumes:
+      - ./db_config/test/server.crt:/var/lib/postgresql/server.crt:ro
+      - ./db_config/test/server.key:/var/lib/postgresql/server.key:ro
```

Comment:
I think these permission descriptors (`:ro`) are not enough.
```
FATAL:  private key file "/var/lib/postgresql/server.key" has group or world access
```

I had to manually set the needed chmod permissions (`0640`) in my machine to make this work. It seems it is consistent with the way the volume permissions work: https://stackoverflow.com/a/50325808

If we can't find a better workaround for this, I think we would need to document this required step in the project `README.md` file.

Reply:
Good catch. I think it should be on whoever is running the repo to manage the correct permissions of the files (while we document on how to do so). R/W permissions for the only only are enough according to https://www.postgresql.org/docs/current/ssl-tcp.html#SSL-SETUP. Documented the steps in. [765de50](https://github.com/safe-global/safe-client-gateway/pull/1235/commits/765de5047d9dced2c9b007dc2c015a71ba9a4423)